src/Table.luna

```diff
@@ -152,21 +395,78 @@ class Table:
     def sort sortBy:
         sortByColumns = sortBy.each (colName, order, pos): (self.column colName, order, pos)
         self.sortByColumn sortByColumns
+
+    # Remove rows with missing values.
+    #
+    # > import Dataframes.Table
+    # > import Dataframes.IO 
+    # > table = CSVParser.readFile "./data/simple_empty.csv"
+    # > newTable = table.dropNa
+    #
+    # `return`: `Table` value without rows containing NA records.
+
     def dropNa:
         self.fromWrapper $ self.ptr.dropNa
+
+    # Remove rows where missing values occurs for given column.
+    #
+    # > import Dataframes.Table
+    # > import Dataframes.IO 
+    # > table = CSVParser.readFile "./data/simple_empty.csv"
+    # > newTable = table.dropNaAt "b"
+    #
+    # `name`: Column name for which we check NA values.
+    #
+    # `return`: `Table` value without rows containing NA records in given column.
```

Comment:
not sure about "record" usage here

Reply:
So how would we name single cell value in table? Any ideas? @piotrMocz @mikusp